numerapi/utils.py

```diff
@@ -34,15 +34,32 @@ def replace(dictionary, key, function):
 
 
 def download_file(url, dest_path, show_progress_bars=True):
+    file_size = 0
     r = requests.get(url, stream=True)
     r.raise_for_status()
     # Total size in bytes.
     total_size = int(r.headers.get('content-length', 0))
-
+    
+    if os.path.exists(dest_path):
+        file_size = os.stat(dest_path).st_size # File size in bytes
+        if file_size < total_size:
+            resume_header = {'Range': 'bytes=%d-' % file_size}
+            r = requests.get(url, headers=resume_header, stream=True,  verify=False, allow_redirects=True)
```

Comment:
please run 'pycodestyle' (and  optionally 'pylint') and fix all warnings the linter gives you